src/service.py

```diff
@@ -50,11 +70,54 @@ def update_project_details_(project: Projects, project_data: Project, db: Sessio
         project.name = project_data.name
     if project_data.description:
         project.description = project_data.description
-
     db.commit()
     db.refresh(project)
 
 
 def delete_project_(project: Projects, db: Session) -> None:
+    db.query(UserProject).filter(UserProject.project_id == project.id).delete()
     db.delete(project)
     db.commit()
+
+
+def create_user_(db: Session, create_user_request: User) -> None:
+    h = hashlib.new("SHA256")
+    h.update(create_user_request.password.encode())
+    password_hash = h.hexdigest()
```

Comment:
you shouldn't have this on a db level function. This info must come to here in a create model already